src/Symfony/Component/Notifier/Bridge/LineBot/LineBotTransport.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotTransport extends AbstractTransport
+{
+    protected const HOST = 'api.line.me';
+
+    public function __construct(
+        #[\SensitiveParameter] private readonly string $channelAccessToken,
+        private readonly string $receiver,
+        ?HttpClientInterface $client = null,
+        ?EventDispatcherInterface $dispatcher = null,
+    ) {
+        parent::__construct($client, $dispatcher);
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof ChatMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
+        }
+        if (null === $this->client) {
+            throw new \LogicException('The LineBot transport cannot send a message without an HTTP client.');
+        }
+
+        $response = $this->client->request(
+            'POST',
+            "https://{$this->getEndpoint()}/v2/bot/message/push",
+            [
+                'headers' => [
+                    'Authorization' => "Bearer $this->channelAccessToken",
+                ],
```

Comment:
Symfony http client has `auth_bearer` key so simplify this

Reply:
Sure. I concatenate by myself because I used to encounter the error `Invalid character found in option 'auth_bearer'`. It seems this was due to other bugs in my bridge, and I no longer see that error now.